src/sass/claims-status.scss

```diff
@@ -218,127 +208,121 @@ a.claim-list-item {
   }
 }
 
-.claims-status {
-  i.fa {
-    font-size: 24px;
-  }
+.claims-status-icon {
+  font-size: 24px;
+}
 
-  .next-action-container {
-    padding-right: 1em;
-  }
+.next-action-container {
+  padding-right: 1em;
+}
 
-  .next-action {
-    padding: 1em;
-    position: relative;
+.next-action {
+  padding: 1em;
+  position: relative;
 
-    p {
-      margin-top: 0;
-    }
+  &.hearing {
+    background: $color-inset-bg;
+  }
 
-    &.hearing {
-      background: $color-inset-bg;
-    }
+  &.soc, &.ssoc {
+    background: $color-gibill-accent;
+    padding-left: 2.5em;
 
-    &.soc, &.ssoc {
-      background: $color-gibill-accent;
-      padding-left: 2.5em;
-
-      &:before {
-        left: 1em;
-        top: 1.5em;
-        position: absolute;
-        font-size: 1.25em;
-        display: inline-block;
-        font: normal normal normal 14px/1 FontAwesome;
-        font-size: inherit;
-        text-rendering: auto;
-        -webkit-font-smoothing: antialiased;
-        -moz-osx-font-smoothing: grayscale;
-        content: "\F071";
-      }
+    &:before {
+      left: 1em;
+      top: 1.5em;
+      position: absolute;
+      font-size: 1.25em;
+      display: inline-block;
+      font: normal normal normal 14px/1 FontAwesome;
+      font-size: inherit;
+      text-rendering: auto;
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+      content: "\F071";
     }
   }
+}
 
-  .last-status {
-    margin-top: 1em;
-    padding: 0 1em 1em 0;
+.next-action-text {
+  margin-top: 0;
+}
 
-    h4:first-of-type {
-      padding: 0;
-    }
+.last-status {
+  margin-top: 1em;
+  padding: 0 1em 1em 0;
+}
 
-    .content {
-      position: relative;
-      padding-left: 3.25rem;
+.content {
+  position: relative;
+  padding-left: 3.25rem;
+}
 
-      h5 + p {
-        margin-top: 0;
-      }
+h5 + .content-text {
```

Comment:
I don't think this is actually used. The h5 has date content between it and the content

Reply:
Good point, plus this should be moved into the status directly, rather than assigned at invocation, as you mentioned elsewhere.

Reply:
There was one instance where this was used (for the `bva_final_decision` status description), but it can be specified directly with the new `description-text` (margin-top: 0) class.